src/components/account/Profile.tsx

```
@@ -0,0 +1,237 @@
+        {isEditing ? (
+          <EditProfile user={user} setIsEditing={setIsEditing} />
+        ) : (
+          <div className="bg-white shadow overflow-hidden sm:rounded-lg dark:bg-slate-700">
+            <div className="px-4 py-5 sm:px-6 flex justify-between items-center">
+              <h3 className="text-lg leading-6 font-medium text-gray-900 dark:text-gray-200">
+                User Profile
+              </h3>
+              <button
+                onClick={() => setIsEditing(true)}
+                className="inline-flex items-center px-3 py-2 border border-transparent text-sm leading-4 font-medium rounded-md text-white bg-indigo-600 hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500"
+              >
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)